misk/src/main/kotlin/misk/web/actions/DashboardMetadataAction.kt

```diff
@@ -36,13 +38,55 @@ class DashboardMetadataAction @Inject constructor() : WebAction {
     @PathParam dashboard: String
   ): Response {
     val caller = callerProvider.get() ?: return Response()
-    val dashboardTabs = allDashboardTabs.filter { it.dashboard == dashboard }
+
+    val dashboardTabs = allTabs.filter { it.dashboard == dashboard }
     val authorizedDashboardTabs =
       dashboardTabs.filter { caller.isAllowed(it.capabilities, it.services) }
-    return Response(tabs = authorizedDashboardTabs)
+
+    val navbarItems = allNavbarItems
+      .filter { it.dashboard == dashboard }
+      .sortedBy { it.order }
+      .map {it.item}
+
+    val navbarStatus = allNavbarStatus.find { it.dashboard == dashboard }?.status ?: ""
```

Comment:
nit: imo, more readable to add a few newlines (e.g., before `.find` and before `.status`